tldw_Server_API/app/core/Notifications/email_delivery.py

```
@@ -0,0 +1,139 @@
+                server.starttls(context=context)
+                if config["user"] and config["password"]:
+                    server.login(str(config["user"]), str(config["password"]))
+                server.sendmail(str(config["from_address"]), to, msg.as_string())
+        else:
+            with smtplib.SMTP(str(config["host"]), int(config["port"])) as server:
+                if config["user"] and config["password"]:
+                    server.login(str(config["user"]), str(config["password"]))
+                server.sendmail(str(config["from_address"]), to, msg.as_string())
+
+        logger.info(f"Notification email sent to {to}: {subject}")
+        return True
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)